api/shopware-6-client.api.md

```diff
@@ -177,6 +177,14 @@ export interface GetNavigationParams {
     rootNode?: string;
 }
 
+// @beta

```

Comment:
so it's marked as a beta :thinking:

Reply:
thanks to this it appears in doc: https://shopware-pwa-docs-git-checkout-payments.shopware-pwa.now.sh/api/shopware-6-client.getorderpaymenturl.html

 we need to mark all of them this way but it's planned in here: https://github.com/DivanteLtd/shopware-pwa/issues/701